lib/membrane_rtc_engine/media_event.ex

```diff
@@ -22,6 +22,11 @@ defmodule Membrane.RTC.Engine.MediaEvent do
     |> do_create(peer_id)
   end
 
+  @spec create_peer_mids_update_event(peer_id_t(), map()) :: sfu_media_event_t()
+  def create_peer_mids_update_event(peer_id, data) do
+    %{type: "updateMid", data: %{id: peer_id, peersInRoom: data}} |> do_create(peer_id)
```

Comment:
I agree that updateMid message should be merged into SDP answer, but I am not sure if peerAccepted should include serverTracks, because this can lead to situation where we will get SDP offer before we get :local_credentials. Adding mapping between track_id and track_metadata is impossible in peerAccepted, because track_id will be generated after you got sdp_offer which is after browser get server_tracks.